cmd/bundle/utils/utils.go

```diff
@@ -19,7 +22,7 @@ func configureVariables(cmd *cobra.Command, b *bundle.Bundle, variables []string
 	})
 }
 
-func ConfigureBundleWithVariables(cmd *cobra.Command) *bundle.Bundle {
+func InitializeBundle(cmd *cobra.Command) *bundle.Bundle {
```

Comment:
This is different from `phases.Initialize()`.

We can rename it to something else, but let's do that outside this PR.